_security/access-control/field-masking.md

```diff
@@ -78,6 +78,16 @@ See [Create role]({{site.url}}{{site.baseurl}}/security/access-control/api/#crea
 
 By default, the Security plugin uses the BLAKE2b algorithm, but you can use any hashing algorithm that your JVM provides. This list typically includes MD5, SHA-1, SHA-384, and SHA-512.
 
+You can override the default algorithm in `opensearch.yml`:
```

Comment:
```suggestion
You can override the default algorithm in `opensearch.yml` using the following setting:
```

Reply:
Thanks @Naarcha-AWS  It doesn't use these additional words in the Salt instructions that I compared with https://opensearch.org/docs/latest/security/access-control/field-masking/#set-the-salt but I can add them if you feel I should.